modular_citadel/code/modules/research/designs/autolathe_designs.dm

```diff
@@ -0,0 +1,15 @@
+/datum/design/hypovialsmall
+	name = "Hypovial"
+	id = "hypovial"
+	build_type = AUTOLATHE
+	materials = list(MAT_METAL = 500)
+	build_path = /obj/item/reagent_containers/glass/bottle/vial/small
+	category = list("initial","Medical")
+
+/datum/design/bucket

```

Comment:
Mister bucket, yer the one~

Reply:
That shouldn't even be able to be there. I copypasted bucket for the small vial, then copypasted the small vial for the big.